demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>..meaning the each embedding.</blockquote><p> </p><p>Re-write</p><p> </p><blockquote>There's many tools...</blockquote><p> </p><p>There are...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>